1800 El ocaso de la humanidad/1800 el ocaso de la humanidad.html

```diff
@@ -0,0 +1,1504 @@
+<div>
+  <img src="https://i.imgur.com/ZVxL8ZG.png" width="800" height="800"  />
+ </div> 
+ <tr> <td colspan="10">&nbsp;</td>
+ </tr>
+<div class="sheet-maindiv">
+<div class="sheet-2colrow">
+   <div class="sheet-col">
+      <table>
```

Comment:
New sheets looking to be added to the repository are required to meet the [Building Character Sheet](https://wiki.roll20.net/Building_Character_Sheets) standards. In particular character sheets should not use `<table>` for layouts. Older sheets in the past have used this method but it is no longer accepted.